developer-docs-site/docs/concepts/state-sync.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "State Synchronization"
+slug: "state-sync"
+---
+
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# State Synchronization
+
+Nodes in an Aptos network, both the validator nodes and the fullnodes, must always be synchronized to the latest Aptos blockchain state. The state synchronization (state sync) component that runs on each node is responsible for this synchronization. To achieve this synchronization, the state sync identifies and fetches new blockchain data from the peers, validates the data and persists it to the local storage.
```

Comment:
nit:
"To achieve this synchronization, the state sync identifies" -> "To achieve this synchronization, state sync identifies"